h2o-algos/src/main/java/hex/schemas/GBMV3.java

```diff
@@ -97,6 +99,11 @@ public static final class GBMParametersV3 extends SharedTreeV3.SharedTreeParamet
     @API(help="A set of allowed column interactions.", level= API.Level.expert)
     public String[][] interaction_constraints;
 
+    @API(help="In-training checkpoints", level = API.Level.expert, gridable = false)
```

Comment:
Nitpicking: this help message will be the description offered to the user in Flow: shouldn't it be a bit more explanatory? short description of what in-training checkpoints are? (not obvious to me)

Reply:
Done:

> Create checkpoints into defined directory while training process is still running. In case of cluster shutdown, this checkpoint can be used to restart training.